main.js

```diff
@@ -7,120 +7,124 @@ const { addExtra } = require("puppeteer-extra");
+const { getAllCoordinates } = require("./getGeocode");
 
 //aws-sdk is only a dev dependency because Lambda already includes the package by default.
 const AWS = require("aws-sdk");
 
 async function execute() {
-	//S3 bucket initialization
-	const s3 = new AWS.S3({
-		accessKeyId: process.env.AWSACCESSKEYID,
-		secretAccessKey: process.env.AWSSECRETACCESSKEY,
-	});
+  //S3 bucket initialization
+  const s3 = new AWS.S3({
+    accessKeyId: process.env.AWSACCESSKEYID,
+    secretAccessKey: process.env.AWSSECRETACCESSKEY,
+  });
 
-	Puppeteer.use(
-		Recaptcha({
-			provider: { id: "2captcha", token: process.env.RECAPTCHATOKEN },
-		})
-	);
+  Puppeteer.use(
+    Recaptcha({
+      provider: { id: "2captcha", token: process.env.RECAPTCHATOKEN },
+    })
+  );
 
-	const browser = process.env.DEVELOPMENT
-		? await Puppeteer.launch({
-				executablePath: process.env.CHROMEPATH,
-		  })
-		: await Puppeteer.launch({
-				args: chromium.args,
-				defaultViewport: chromium.defaultViewport,
-				executablePath: await chromium.executablePath,
-				headless: chromium.headless,
-				ignoreHTTPSErrors: true,
-		  });
+  const browser = process.env.DEVELOPMENT
+    ? await Puppeteer.launch({
+        executablePath: process.env.CHROMEPATH,
+      })
+    : await Puppeteer.launch({
+        args: chromium.args,
+        defaultViewport: chromium.defaultViewport,
+        executablePath: await chromium.executablePath,
+        headless: chromium.headless,
+        ignoreHTTPSErrors: true,
+      });
 
-	const gatherData = async () => {
-		const results = await Promise.all(
-			scrapers.map((scraper) =>
-				scraper(browser).catch((error) => {
-					//print out the issue but don't fail, this way we still publish updates
-					//for other locations even if this website's scrape doesn't work
-					console.log(error);
-					return null;
-				})
-			)
-		);
-		browser.close();
-		const finalResultsArray = [];
-		for (const result of results) {
-			if (Array.isArray(result)) {
-				finalResultsArray.push(...result);
-			} else if (result) {
-				//ignore nulls
-				finalResultsArray.push(result);
-			}
-		}
+  const gatherData = async () => {
+    const results = await Promise.all(
+      scrapers.map((scraper) =>
+        scraper(browser).catch((error) => {
+          //print out the issue but don't fail, this way we still publish updates
+          //for other locations even if this website's scrape doesn't work
+          console.log(error);
+          return null;
+        })
+      )
+    );
+    browser.close();
+    const finalResultsArray = [];
+    for (const result of results) {
+      if (Array.isArray(result)) {
+        finalResultsArray.push(...result);
+      } else if (result) {
+        //ignore nulls
+        finalResultsArray.push(result);
+      }
+    }
 
-		const responseJson = {
-			results: finalResultsArray,
-		};
+    const responseJson = {
+      results: await getAllCoordinates(finalResultsArray),
```

Comment:
nit: maybe we should rename `finalResultsArray` since it's not really final if we have to get all the coords still